### PR TITLE
fix broken Accoutrement links

### PR DESCRIPTION
**Pull request by dvdherron**
## Steps to test/reproduce
See the list of Accoutrement modules at `/accoutrement/`
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me
_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Comment by dvdherron** [comment]
[Since we updated Accoutrement, we probably should remove the links and descriptions for Core and Init and add Tokens, Ratios, Utilities?](https://trello.com/c/oZDrk9MN/162-since-we-updated-accoutrement-we-probably-should-remove-the-links-and-descriptions-for-core-and-init-and-add-tokens-ratios-utili)

**Comment by dvdherron** [comment]
I removed links and mentions to "Init" and "Core" on the Accoutrement page and added the "Tokens", "Ratios", and "Utilities" modules. 

Should we reuse the same icon (core) for these added modules? Or are there more of these icons to choose from someplace else? @stacyk @mirisuzanne 

image

**Review by jgerigmeyer — APPROVED** [review]
I notice that this list is slightly different than the list of modules on https://www.oddbird.net/accoutrement/docs/ but that might be intentional.

**Comment by dvdherron** [comment]
> I notice that this list is slightly different than the list of modules on https://www.oddbird.net/accoutrement/docs/ but that might be intentional.

Hmm, yeah, these lists are slightly different. I added "Variables" to the list on OddSite. I'm guessing it would make sense to  add "Layout" to the list of "Modules for common data types" in the Accoutrement docs too  @mirisuzanne ?

 (Tokens is not in that list in the docs but is described in the following section)

**Comment by mirisuzanne** [comment]
I think it would make most sense for both these lists to match the top-level headings in the Accoutrement docs sidebar navigation. With the possible special-casing of Tokens in the readme, as that's a meta-syntax that any of the other modules can rely on.

The icons are all from linearicons set, which we have in the company dropbox. I don't know if we still feel like there are icons that make sense for every module, or if we want to reconsider those. I think we either want to find a unique icon for each module, or reconsider those icons -- it's a bit strange to have only some of them repeated.

**Comment by dvdherron** [comment]
> I think it would make most sense for both these lists to match the top-level headings in the Accoutrement docs sidebar navigation. With the possible special-casing of Tokens in the readme, as that's a meta-syntax that any of the other modules can rely on.

Makes sense. I edited this list to match the Accoutrement docs sidebar navigation. And I'll take care of the list on the docs page as well. 

> 
> The icons are all from linearicons set, which we have in the company dropbox. I don't know if we still feel like there are icons that make sense for every module, or if we want to reconsider those. I think we either want to find a unique icon for each module, or reconsider those icons -- it's a bit strange to have only some of them repeated.

Added new icons for Tokens and Ratios. @stacyk Can you tell why the outer circle path isn't showing up on the two icons I've added? Changing the viewbox to match the other icons makes the svg show up as a tiny pixels. 

image

File: content/_includes/logos/accoutrement_tokens.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<svg
```
[the remark this answers is not in the document]
Comment (dvdherron) [reply content/_includes/logos/accoutrement_tokens.svg:1]: @mirisuzanne @stacyk 

Some other possibilities for "Tokens"

key-value pairs
  - key
  
image

- keyhole

image

Searching

- flashlight

image

- binoculars

image

Dictionary

- book

image

[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/logos/accoutrement_tokens.svg:1]: I kinda like the book.

[the remark this answers is not in the document]
Comment (stacyk) [reply content/_includes/logos/accoutrement_tokens.svg:1]: And I'm team key. But I don't feel strongly about it

[the remark this answers is not in the document]
Comment (stacyk) [reply content/_includes/logos/accoutrement_tokens.svg:1]: We could do something like building block-esque 

https://thenounproject.com/icon/modules-2209534/
Screen Shot 2022-03-24 at 12 30 08 PM
 
or tetris pieces
Screen Shot 2022-03-24 at 12 31 45 PM

https://thenounproject.com/icon/tetris-piece-689980/
https://thenounproject.com/icon/tetris-piece-3644812/

But I still like the simplicity of the key as well.

**Comment by stacyk** [comment]
> Added new icons for Tokens and Ratios. @stacyk Can you tell why the outer circle path isn't showing up on the two icons I've added? Changing the viewbox to match the other icons makes the svg show up as a tiny pixels.
> 

@dvdherron The path for that circle is way outside the viewbox. I feel like we probably should have a stroke on these instead of a path, so they could all be consistent, but sometimes having more control over the elements is nice. I'll update the ratio icon for now but hold on that token one. 

Screen Shot 2022-03-24 at 12 39 39 PM

**Comment by dvdherron** [comment]
> > Added new icons for Tokens and Ratios. @stacyk Can you tell why the outer circle path isn't showing up on the two icons I've added? Changing the viewbox to match the other icons makes the svg show up as a tiny pixels.
> 
> @dvdherron The path for that circle is way outside the viewbox. I feel like we probably should have a stroke on these instead of a path, so they could all be consistent, but sometimes having more control over the elements is nice. I'll update the ratio icon for now but hold on that token one.
> 
> Screen Shot 2022-03-24 at 12 39 39 PM

Aha. Yeah, I noticed that and realized it needed to be resized and centered but couldn't get it to sit like the others. I think I'm not using the right tool to edit these. Maybe an SVG crash course? 

And as far as key versus book versus blocks. .  .I probably lean towards key or book more but I don't have a strong feeling here either @stacyk

**Comment by stacyk** [comment]
@dvdherron I just pushed a fix. SVG is easiest to deal with in a program like Illustrator. You can see everything there, whether it is in frame or not. I'd be happy to do a walk through for anyone who'd like it.

File: content/_includes/logos/accoutrement_tokens.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/_includes/logos/accoutrement_tokens.svg:1]: I can get behind the key :)

[the remark this answers is not in the document]
Comment (dvdherron) [reply content/_includes/logos/accoutrement_tokens.svg:1]: Key it is @mirisuzanne @stacyk 

Added the key icon in [e667273](https://github.com/oddbird/oddleventy/pull/201/commits/e6672730e913e0329addf6b0bf01360dbfe14f31)

bild

**Review by stacyk — APPROVED** [review]
LGTM!

**Review by jgerigmeyer — APPROVED** [review]

**Review by mirisuzanne — APPROVED** [review]

**Pull request merged**